server/utils/utils.ts

```diff
@@ -94,3 +94,24 @@ export const formatMoney = (val: number, emptyState: string = undefined, currenc
 export const formatPrivilegedVisitsSummary = (count: number): string => {
   return `Including ${count} privileged visits`
 }
+
+export const arrayToQueryString = (array: string[] | number[] | boolean[], key: string): string =>
+  array && array.map(item => `${key}=${encodeURIComponent(item)}`).join('&')
+
+export const mapToQueryString = (params: Record<never, never>): string =>
+  Object.keys(params)
+    .filter(key => params[key])
+    .map(key => {
+      if (Array.isArray(params[key])) return arrayToQueryString(params[key], key)
+      return `${key}=${encodeURIComponent(params[key])}`
+    })
+    .join('&')
+
+export const getNamesFromString = (string: string): string[] =>
+  string &&
+  string
+    .split(', ')
+    .reverse()
+    .join(' ')
+    .split(' ')
+    .map(name => properCaseName(name))
```

Comment:
Similar to my other comment - it'd be good to make sure we have tests around these functions as we'd want to make sure we don't accidentally break any of this if they were to be edited in the future